From c79c43de78210e196e0c053e536e0869916a5f3b Mon Sep 17 00:00:00 2001
From: Arnon Kaufman <akaufman@habana.ai>
Date: Wed, 29 May 2024 17:33:45 +0300
Subject: [PATCH] [SW-15774] board: fix its directories

The preliminary directory set for using by the ITS template formatting
was wrong and needed an adjustment

Change-Id: Ie28f24ee296b02158e0dc8f9fba9a6a97925af4b
Signed-off-by: Arnon Kaufman <akaufman@habana.ai>
---

diff --git a/board/habanalabs/common/agent-fw_loader.its b/board/habanalabs/common/agent-fw_loader.its
index 795c2e4..6080cf0 100644
--- a/board/habanalabs/common/agent-fw_loader.its
+++ b/board/habanalabs/common/agent-fw_loader.its
@@ -16,7 +16,7 @@
 	images {
 		security {
 			description = "Habana security image";
-			data = /incbin/("@FILES_PATH_1@@DEV_PREFIX@-security-arc.bin.@ENDIANNESS@");
+			data = /incbin/("@FILES_PATH_2@@DEV_PREFIX@-security-arc.bin.@ENDIANNESS@");
 			compression = "none";
 			offset = <@SEC_OFFSET@>;
 			hash {
@@ -26,7 +26,7 @@
 
 		img-ptr {
 			description = "Habana image pointers";
-			data = /incbin/("@FILES_PATH_1@@DEV_PREFIX@-@IMG_BIN@.bin.@ENDIANNESS@");
+			data = /incbin/("@FILES_PATH_2@@DEV_PREFIX@-@IMG_BIN@.bin.@ENDIANNESS@");
 			compression = "none";
 			offset = <@IMG_PTR_OFFSET@>;
 			hash {
@@ -36,7 +36,7 @@
 
 		ppboot {
 			description = "Habana PPboot";
-			data = /incbin/("@FILES_PATH_1@@DEV_PREFIX@-@PPBOOT_BIN@.bin.@ENDIANNESS@");
+			data = /incbin/("@FILES_PATH_2@@DEV_PREFIX@-@PPBOOT_BIN@.bin.@ENDIANNESS@");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
@@ -48,7 +48,7 @@
 
 		preboot {
 			description = "Habana Preboot";
-			data = /incbin/("@FILES_PATH_1@@DEV_PREFIX@-@PREBOOT_BIN@.bin.@ENDIANNESS@");
+			data = /incbin/("@FILES_PATH_2@@DEV_PREFIX@-@PREBOOT_BIN@.bin.@ENDIANNESS@");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
diff --git a/board/habanalabs/common/agent-fw_loader_erom.its b/board/habanalabs/common/agent-fw_loader_erom.its
index e4fd4a3..55bd6b5 100644
--- a/board/habanalabs/common/agent-fw_loader_erom.its
+++ b/board/habanalabs/common/agent-fw_loader_erom.its
@@ -15,7 +15,7 @@
 	images {
 		erom {
 			description = "Erom";
-			data = /incbin/("@FILES_PATH_1@@HL_DEV@-erom.bin.@ENDIANNESS@");
+			data = /incbin/("@FILES_PATH_2@@HL_DEV@-erom.bin.@ENDIANNESS@");
 			type = "standalone";
 			compression = "none";
 			offset = <0>;
diff --git a/board/habanalabs/common/spi-fpga-secureboot.its b/board/habanalabs/common/spi-fpga-secureboot.its
index f0681ce..8d4b597 100644
--- a/board/habanalabs/common/spi-fpga-secureboot.its
+++ b/board/habanalabs/common/spi-fpga-secureboot.its
@@ -11,7 +11,7 @@
 	images {
 		security {
 			description = "Habana ARC security";
-			data = /incbin/("@FILES_PATH_1@@HL_DEV@-security-arc.bin.le");
+			data = /incbin/("@FILES_PATH_2@@HL_DEV@-security-arc.bin.le");
 			compression = "none";
 			offset = <0x00010000>;
 			hash {
@@ -21,7 +21,7 @@
 
 		preboot {
 			description = "Habana Preboot";
-			data = /incbin/("@FILES_PATH_1@@HL_DEV@-preboot.bin.le");
+			data = /incbin/("@FILES_PATH_2@@HL_DEV@-preboot.bin.le");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
@@ -35,7 +35,7 @@
 
 		ppboot {
 			description = "Habana PPboot";
-			data = /incbin/("@FILES_PATH_1@@HL_DEV@-ppboot-pcie_phy_fw.bin.le");
+			data = /incbin/("@FILES_PATH_2@@HL_DEV@-ppboot-pcie_phy_fw.bin.le");
 			type = "standalone";
 			arch = "arc";
 			compression = "none";
@@ -49,7 +49,7 @@
 
 		img-ptr {
 			description = "Habana image pointers";
-			data = /incbin/("@FILES_PATH_1@@HL_DEV@-images-pointers.bin.le");
+			data = /incbin/("@FILES_PATH_2@@HL_DEV@-images-pointers.bin.le");
 			compression = "none";
 			offset = <0x00060000>;
 			hash {
diff --git a/board/habanalabs/common/zephyr.its b/board/habanalabs/common/zephyr.its
index f6c82d8..0a7ad5e 100644
--- a/board/habanalabs/common/zephyr.its
+++ b/board/habanalabs/common/zephyr.its
@@ -9,7 +9,7 @@
 	images {
 		zephyr-pid-iccm {
 			description = "Zephyr pid ICCM";
-			data = /incbin/("@FILES_PATH_1@@DEV_NAME@-zephyr-pid-iccm.bin.le");
+			data = /incbin/("@FILES_PATH_2@@DEV_NAME@-zephyr-pid-iccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_D0_PSOC_ARC0@>;
@@ -23,7 +23,7 @@
 
 		zephyr-pid-dccm {
 			description = "Zephyr pid DCCM";
-			data = /incbin/("@FILES_PATH_1@@DEV_NAME@-zephyr-pid-dccm.bin.le");
+			data = /incbin/("@FILES_PATH_2@@DEV_NAME@-zephyr-pid-dccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_D0_PSOC_ARC0@>;
@@ -36,7 +36,7 @@
 
 		zephyr-mgmt-iccm {
 			description = "Zephyr mgmt ICCM";
-			data = /incbin/("@FILES_PATH_1@@DEV_NAME@-zephyr-mgmt-iccm.bin.le");
+			data = /incbin/("@FILES_PATH_2@@DEV_NAME@-zephyr-mgmt-iccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_D0_PSOC_ARC1@>;
@@ -50,7 +50,7 @@
 
 		zephyr-mgmt-dccm {
 			description = "Zephyr mgmt DCCM";
-			data = /incbin/("@FILES_PATH_1@@DEV_NAME@-zephyr-mgmt-dccm.bin.le");
+			data = /incbin/("@FILES_PATH_2@@DEV_NAME@-zephyr-mgmt-dccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_D0_PSOC_ARC1@>;
@@ -63,7 +63,7 @@
 
 		zephyr-event-iccm {
 			description = "Zephyr event ICCM";
-			data = /incbin/("@FILES_PATH_1@@DEV_NAME@-zephyr-event-iccm.bin.le");
+			data = /incbin/("@FILES_PATH_2@@DEV_NAME@-zephyr-event-iccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_D0_PSOC_ARC2@>;
@@ -77,7 +77,7 @@
 
 		zephyr-event-dccm {
 			description = "Zephyr event DCCM";
-			data = /incbin/("@FILES_PATH_1@@DEV_NAME@-zephyr-event-dccm.bin.le");
+			data = /incbin/("@FILES_PATH_2@@DEV_NAME@-zephyr-event-dccm.bin.le");
 			type = "standalone";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_D0_PSOC_ARC2@>;
@@ -90,7 +90,7 @@
 
 		external_fw {
 			description = "NIC PHY FW image";
-			data = /incbin/("@FILES_PATH_1@@DEV_NAME@-eth-phy.bin.le");
+			data = /incbin/("@FILES_PATH_2@@DEV_NAME@-eth-phy.bin.le");
 			compression = "none";
 			arch = "arc";
 			core-id = <@FIT_CORE_ID_D0_PSOC_ARC1@>;
